src/App.js

```diff
@@ -24,24 +24,11 @@ const alex = {
 };
 
 const App = () => (
-  <div className="App">
-    <section className="Person">
-      <h2 className="Person__name">My name is Misha</h2>
-      <p className="Person__age">I am 37</p>
-      <p className="Person__partner">My wife&apos;s name is Natasha</p>
-    </section>
-
-    <section className="Person">
-      <h2 className="Person__name">My name is Olya</h2>
-      <p className="Person__partner">My husband&apos;s name is Maksym</p>
-    </section>
-
-    <section className="Person">
-      <h2 className="Person__name">My name is Alex</h2>
-      <p className="Person__age">I am 25</p>
-      <p className="Person__partner">I am not married</p>
-    </section>
-  </div>
+  <>
+    <Person {...misha} />
+    <Person {...olya} />
+    <Person {...alex} />
+  </>
```

Comment:
React Fragment is redundant here
```suggestion
  <Person {...misha} />
  <Person {...olya} />
  <Person {...alex} />
```